app/ante.go

```diff
@@ -29,6 +34,8 @@ type HandlerOptions struct {
 	IBCKeeper     *keeper.Keeper
 	CircuitKeeper *circuitkeeper.Keeper
 	RateMinMax    RateMinMax
+	WasmConfig    *wasmtypes.WasmConfig
+	StoreKey      *types.KVStoreKey
```

Comment:
storetypes "github.com/cosmos/cosmos-sdk/store/types"
TxCounterStoreKey storetypes.StoreKey

...
wasmkeeper.NewCountTXDecorator(opts.TxCounterStoreKey),

this shouldn't be a pointer

Reply:
Fixed in 08fab80995af1176d8fecf9f66110c323d5a7075